.github/workflows/update-weblate.yml

```diff
@@ -13,6 +13,16 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    # If an open Weblate/translations PR still exists, we shouldn't tell Weblate to push again.
+    # Because we ended our previous push with a 'reset', Weblate would otherwise force-push
+    # new changes over the unmerged old changes, causing them to lost.
+    - name: Check for existence of old Weblate PR
+      run: |
+        if [[ "$(gh pr -R hedyorg/hedy list --search 'Translations update' --json number)" != "[]" ]]; then
```

Comment:
Searching for 'Translations update' returns the current PR, which I think is not the indent.
<img width="861" alt="Screenshot 2024-04-11 at 14 39 08" src="https://github.com/hedyorg/hedy/assets/3010723/3bc425b6-25e9-4e4a-b43a-531143cc94df">

The issue is addressed if I add double quotes in the search parameter: `'"Translations update"'`